.github/workflows/publish.yml

```
@@ -39,7 +39,7 @@ jobs:
 
     - name: Publish a Python distribution to PyPI
       if: ${{ github.event_name == 'release' }}
-      uses: pypa/gh-action-pypi-publish@master
+      uses: pypa/gh-action-pypi-publish@v1
```

I believe we need to pin this one to the exact version unlike the others.

```suggestion
      uses: pypa/gh-action-pypi-publish@v1.5.1
```